Gems/ROS2/Code/Include/ROS2/Manipulation/JointsManipulationRequests.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/ComponentBus.h>
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/std/containers/vector.h>
+#include <AzCore/std/string/string.h>
+#include <AzCore/std/utils.h>
```

Comment:
I'm not sure why ComponentBus is brought in here but not used?  If you want to make `JointsManipulationRequests` inherit from `ComponentBus` you shouldn't need to state the `BusIdType` and `AddressPolicy` but, if you don't want to inherit from `ComponentBus` then you can probably remove the `ComponentBus.h` include.

```suggestion
#include <AzCore/Component/ComponentBus.h>
#include <AzCore/std/containers/vector.h>
#include <AzCore/std/string/string.h>
```

Reply:
EntityComponentIdPair is defined in ComponentBus.h. This header should be in JointInfo.h though.